src/handlers.ts

```diff
@@ -61,6 +61,20 @@ export const takerRequestHandler = async (
     return result.end();
 };
 
+export const fetchOtcQuoteHandler = async (quoter: Quoter, req: express.Request, res: express.Response) => {
+    const takerRequestResponse = parseTakerRequest(req);
+
+    if (!takerRequestResponse.isValid) {
+        return res.status(HttpStatus.BAD_REQUEST).json({ errors: takerRequestResponse.errors });
+    }
+
+    const takerRequest = takerRequestResponse.takerRequest;
+
+    const response = await quoter.fetchFirmOtcQuoteAsync(takerRequest);
+    const result = response.order ? res.status(HttpStatus.OK).json(response) : res.status(HttpStatus.NO_CONTENT);
```

Comment:
Are you sure this works?

I thought [`res.json`](http://expressjs.com/en/5x/api.html#res.json) sent the response off. I would think that `result.end` on the next line would cause a response modified after send error.

Reply:
Ok i thought the .end would throw an error or at least warn if the response had already been sent.